temporian/implementation/numpy_cc/operators/tick_calendar.cc

```diff
@@ -0,0 +1,111 @@
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+
+#include <chrono>
+#include <cstdint>
+#include <ctime>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "temporian/implementation/numpy_cc/operators/common.h"
+
+namespace {
+namespace py = pybind11;
+
+py::array_t<double> tick_calendar(
+    const long start_timestamp,                  // min date
```

Comment:
const double

Then, when converting to "std::time_t", apply a floor for begin, and a ceil for end.

Note: "long" is 32 bits (or at least, it can be).